content/docs/use-cases/ml-model-registries.md

```diff
@@ -0,0 +1,54 @@
+# Machine Learning Model Registries
+
+An **ML model registry** (or _model catalog_) works as a central repository
+between data science projects and their applications. ML models and other data
+artifacts can be browsed, shared, audited, tested, and deployed from here. DVC
+repositories support all of this while integrating seamlessly with existing
+development workflows. Other tools from [Iterative](https://iterative.ai/)
+further improve your model catalogs with more features and better usability.
+
+![](/img/ml_model_registry_placement.jpg) _Model registry placement contrasts
+with general [data registries] for reusable data sources or training sets._
+
+<!-- TODO: Look up package registry diagrams for inspiration -->
+
+[data registries]: /doc/use-cases/data-registries
+
+DVC model registries provide your team with key collaborative capabilities:
+
+- Store, organize, and track the versions of different models effectively.
+- Full data provenance and lineage trails guarantee that your modeling is
+  reproducible.
+- Share and control access to early-stage [experiments] and production-ready
+  models alike.
+- Save performance [metrics and plots] to quickly evaluate and compare models.
+- A platform to export, integrate, deploy, or roll-back published models
+
+[experiments]: /doc/user-guide/experiment-management
+[metrics and plots]: /doc/start/metrics-parameters-plots
+
+Many of these benefits are built-into DVC: Your [data pipelines] and resulting
+artifacts (including ML models and performance metadata) are **codified** into
+one or mode <abbr>DVC projects</abbr>. This enables managing them with standard
+Git [versioning] workflows. Large files can be pushed to [remote storage], an
+efficient and scalable access point for your models.
+
+To achieve the other goals, the final outputs of these model development
+projects can be **packaged** and published in a global DVC repository. Existing
+model files stored remotely are reused in this collection without copying them,
+so building the registry is fast. Links to the original projects are preserved,
+so you can always retrieve full lineage and provenance info.
+
+Distributed DVC repos and managed storage are already great for sharing data and
+models. And with a model registry, getting them to production is also
+straightforward: Use the DVC CLI to [import] them for deployment, integrate them
```

Comment:
it can be also DVC Python API, and probably `dvc get` rather than import

Reply:
> it can be also DVC Python API

That's already in the text: "integrate them directly into code with the Python API" - changed to "DVC's Python API" to make it more explicit.

> probably dvc get rather than import

Good point, updated.